README.rst

```diff
@@ -38,21 +38,21 @@ A node object represents a node in a graph.
 
 **node properties**
 
-- An `id` property is a primary key for an object (see Objects_) that is unique for the object type. Its value is defined as a *JSON string* for flexibility.
+- Each `node` object must contain a string key which is the unique id for the node
```

Comment:
It's unclear what the "string key" is here. Is it still the `id` property?

Reply:
The node id became the node object key (string-value only).